app/src/components/modal/modal_airdrop/airdrop_card.tsx

```diff
@@ -0,0 +1,159 @@
+import { BigNumber } from 'ethers/utils'
+import React, { HTMLAttributes, useEffect, useState } from 'react'
+import styled, { withTheme } from 'styled-components'
+
+import { STANDARD_DECIMALS } from '../../../common/constants'
+import { useAirdropService, useConnectedWeb3Context } from '../../../hooks'
+import { formatBigNumber } from '../../../util/tools'
+import { Button } from '../../button'
+import { ButtonType } from '../../button/button_styling_types'
+import { IconOmen } from '../../common/icons'
+import { BalanceSection, ModalCard } from '../common_styled'
+
+const TopSection = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 16px 20px;
+  width: 100%;
+  border-bottom: ${props => props.theme.borders.borderLineDisabled};
+`
+
+const TopSectionLeft = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`
+
+const TopSectionDetails = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  margin-left: 16px;
+`
+
+const TopSectionHeading = styled.div`
+  color: ${props => props.theme.colors.textColorDark};
+  font-weight: 500;
+`
+
+interface Subheading {
+  green: boolean
+}
+
+const TopSectionSubHeading = styled.div<Subheading>`
+  color: ${props => (props.green ? props.theme.colors.green : props.theme.colors.textColorLighter)};
+  font-weight: 500;
+`
+
+const BottomSection = styled(BalanceSection as any)`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: row;
+`
+
+const BottomSectionTextWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+`
+
+const BottomSectionHeading = styled.div`
+  color: ${props => props.theme.colors.textColorDark};
+  font-weight: 500;
+`
+
+const BottomSectionSubheading = styled.div`
+  color: ${props => props.theme.colors.textColorLighter};
+  font-size: 12px;
+`
+
+const BottomSectionDescription = styled.div`
+  color: ${props => props.theme.colors.textColorLighter};
+`
+
+const AirdropButton = styled(Button)`
+  width: 75px;
+  height: 40px;
```

Comment:
@pimato Also I don't think we should define static heights and widths for the buttons. The height is same if we exclude it. But width is different because figma designs have different paddings then our default button styling. In codebase we use padding:12px 17px and in figma its 12px 18.5px which is causing 3px difference.